File: Server/Source/Models/Scrap/Formula1/Formula1StandingScrap.cs
namespace Server.Source.Models.Scrap.Formula1
{
    public class Formula1StandingScrap
    {
        public List<Formula1StandingScrap_Column> Columns { get; set; } = [];
        public List<string> Rows { get; set; } = [];
        public List<List<string>> Points { get; set; } = [];
    }    

    public class Formula1StandingScrap_Column
    {
        public string Name { get; set; } = string.Empty;
        public string Position { get; set; } = string.Empty;
        public string Image { get; set; } = string.Empty;
        public string Abbreviation { get; set; } = string.Empty;
    }
}

File: Client/src/app/source/models/scrap/formula1-standing.scrap.ts
export class Scr_Formula1StandingScrap {
    raceTracks: string[] = [];
    columnLabels?: Scr_Formula1Standing_ColumnLabel[];
    rowLabels?: Scr_Formula1Standing_RowLabel[];
}

export class Scr_Formula1Standing_ColumnLabel {
    position: string = '';
    image: string = '';
    abbreviation: string = '';
    name: string = '';
}

export class Scr_Formula1Standing_RowLabel {
    points: string[] = [];
}